public/class-award-force-sso.php

```diff
@@ -57,17 +57,27 @@ private function getSlug(WP_User $user)
      */
     private function requestSlug(WP_User $user)
     {
+        $response = $this->requestSlugByEmail($user->user_email.$random);
```

Comment:
The `$random` bit looks like testing code. Should it be there?

Reply:
:+1: